requirements.txt

```diff
@@ -1,44 +0,0 @@
-absl-py==0.7.1
```

Comment:
What were the rest of these requirements for? 

Do we need a new `requirements/dev.txt` or something?